docs/JestObjectAPI.md

```diff
@@ -799,6 +799,10 @@ This means, if any timers have been scheduled (but have not yet executed), they
 
 Returns the number of fake timers still left to run.
 
+### `jest.now()`
+
+Returns the time in ms of the current fake clock.
```

Comment:
Add a note this is equivalent to `Date.now()` for non-legacy timers?

Reply:
I guess it *could* be useful for modern timers too if `doNotFake: ['Date', 'performance']` is set?

Reply:
Indeed, that’s a good point. I was also wondering what does it do if called without faking time? Would it be worth mentioning that?

Reply:
Yeah good point - currently, it'll pass through to `now()` in the default (modern) fake timers implementation, which will try to use `this._clock.now`, which will throw as `this._clock` is only initialised by `useFakeTimers()`. Not good!

Other APIs in that file wrap calls in `_checkTimers()` which prints a warning if fake timers aren't active, and then usually return some dummy value.

So I guess two options:
1. Return the real `Date.now()` (and document that behaviour)
2. Warn on the console and return some dummy value, probably `0`

Reply:
Perhaps 1? Since `jest.now()` is equivalent to mocked `Date.now()`, then for me it makes sense to return the real `Date.now()` also in case if time is not faked.